exercises/grade-school/metadata.toml

```diff
@@ -1,3 +1 @@
 blurb = "Given students' names along with the grade that they are in, create a roster for the school."
-source = "A pairing session with Phil Battos at gSchool"
```

Comment:
In line with https://github.com/exercism/problem-specifications/pull/1663 I'd suggest keeping the source and only removing the URL. The source itself hasn't changed.